public/index.html

```diff
@@ -10,6 +10,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+
```

Comment:
Why you add new line?

Reply:
it was I who removed the unnecessary link and there was an empty line